src/main/java/uk/gov/hmcts/reform/migration/repository/CaseLoader.java

```diff
@@ -46,7 +47,21 @@ public List<SscsCaseDetails> findCases() {
         return cases;
     }
 
-    private String decompressAndB64Decode(String b64Compressed) throws IOException {
+    public LinkedHashMap<String, String> findCasesWithHearingID() {
```

Comment:
could we rename this to match what it's doing which is building caseRefToHearingIdMap